viewer/Viewer.h

```diff
@@ -111,23 +125,54 @@ namespace Enki
 		typedef QMapIterator<const std::type_info*, const std::type_info*> ManagedObjectsAliasesMapIterator;
 		ManagedObjectsAliasesMap managedObjectsAliases;
 		
+		typedef std::pair<QString, unsigned int> viewerMessage;
```

Comment:
I do not understand why these messages are related to picking? My understanding is that they belong to another feature.